app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -5,16 +5,20 @@ import PatientDashboard from '../patientDashboard';
 import SearchAndAddOrder from './SearchAndAddOrder';
 import fetchPatientCareSetting from '../../actions/careSetting';
 import { getSettingEncounterType } from '../../actions/settingEncounterType';
+import { getSettingEncounterRole } from '../../actions/settingEncounterRole';
 import imageLoader from '../../../img/loading.gif';
 
 export class OrderEntryPage extends React.Component {
   componentDidMount() {
     this.props.fetchPatientCareSetting();
     this.props.getSettingEncounterType;
+    this.props.getSettingEncounterRole;
```

Comment:
I guess lines 14 & 15 should be flagged by eslint for being statements/expression rather than functions, please look into fixing them if so

Reply:
done